modules/common/adapters/adapter.h

```diff
@@ -90,6 +90,11 @@ class AdapterBase {
    */
   virtual double GetDelayInMs() const = 0;
 
+  /**
+   * @brief Gets message lag since last received message.
+   */
+  virtual double GetLagSinceLastMessage() const = 0;
```

Comment:
How’s this different with GetDelay?

Reply:
Actually the existing GetDelay() is a little weird to me.
It returns the interval between the last two messages. Which means, if a module worked fine but crashed suddenly, GetDelay() will always return a constant and look-good value.

Reply:
Sounds like a good idea, can you just replace the existing GetDelayInMs to fix the weird logic? I'd like to avoid confusingly having two methods getting delay/lag.